guides/common/modules/proc_creating-custom-provisioning-snippets.adoc

```diff
@@ -1,25 +1,24 @@
 [id="Creating_Custom_Provisioning_Snippets_{context}"]
 = Creating custom provisioning snippets
 
-Custom provisioning snippets allow you to execute custom code during host provisioning.
-You can run code before and/or after the provisioning process.
+You can execute custom code before and/or after the host provisioning process.
 
 .Prerequisites
 * Depending on your provisioning template, multiple custom snippet hooks exist which you can use to include custom provisioning snippets.
```

Comment:
I'm not sure what this means. Suggest rewriting it to define "snippet hooks" or leave out that detail.

Reply:
Then maybe you could remove "Depending ..." and change the next sentence to this: 

"Check your provisioning template to ensure that it supports the custom snippets you want to use."

This also sounds more like a prerequisite. When prerequisites start with an explanatory sentence, I always worry that the user will miss the actual prerequisite. :-)

Reply:
Yes, makes sense to me. Reworded prerequisites in both files.